devel/addon/certmanager/install.sh

```diff
@@ -23,7 +23,7 @@ NAMESPACE="${NAMESPACE:-cert-manager}"
 # Release name to use with Helm
 RELEASE_NAME="${RELEASE_NAME:-cert-manager}"
 # Default feature gates to enable
-FEATURE_GATES="${FEATURE_GATES:-ExperimentalCertificateSigningRequestControllers=true}"
+FEATURE_GATES="${FEATURE_GATES:-ExperimentalCertificateSigningRequestControllers=true\\,ExperimentalGatewayAPISupport=true}"
```

Comment:
I didn't understand why we also have to set this in the environment of the test job: https://github.com/jetstack/testing/pull/543. 
Why isn't this change sufficient?
Happy to merge this regardless, just curious.

Reply:
This install script has default values in for ease of testing locally. However they are explicitly set in the config map for the prow job, which will override these

Reply:
But why are they overridden in prow? The prow config always sets these to true anyway AFAICS. 
Are there some prow jobs where these features are *disabled*?

Reply:
Yes, in 1.16 to 1.18 the ExperimentalCertificateSigningRequestControllers is explicitly disabled. However we could put the logic in this script similar to the skip logic.